cg_hermes/config/balsamic.py

```diff
@@ -175,12 +175,12 @@
         "used_by": ["scout"],
     },
     frozenset({"tnscope", "vcf-pass", "annotated-somatic-vcf-pass", "snv"}): {
-        "tags": ["vcf-snv-clinical", "scope", "filtered", "sention"],
+        "tags": ["vcf", "scope", "filtered", "sention"],
```

Comment:
What does `scope` stand for? Is it hermes specific?

Reply:
`scope` was something me, @Mropat and @keyvanelhami came up with when we sat down and went thorough all the files and tags. The tag in balsamic is `tnscope` and we converted that to something more common since it seemed to recur.

Reply:
`tnscope` is the actual name of the variant caller. So we should keep it as such.  `scope` has no meaning otherwise.

Variant callers in Sentieon are:
`tnscope`, `tnsnv`, `tnhaplotyper`, `dnascope`, and `tnhaplotyper2`